cmd/init.go

```diff
@@ -221,4 +229,5 @@ func init() {
 
 	initCmd.Flags().String("cluster-name", "kubefirst", "the cluster name, used to identify resources on cloud provider")
 	initCmd.Flags().String("version-gitops", "main", "version/branch used on git clone")
+	initCmd.Flags().Bool("disable-telemetry", false, "Installer will not send telemetry about this installation")
```

Comment:
what about name it `telemetry`, and have it enabled by default?

user calls `--telemetry=false` when he decides to opt-out telemetry